### PR TITLE
Fix InspectStringMultipleRulesTests to use large enough string to det…

### PR DESCRIPTION
**Pull request by pheonixblade9**
…ect properly

### CODE REVIEW

File: dlp/api/Snippets/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # .NET Cloud Translation API Samples
```
Comment (amanda-tarafa) [inline dlp/api/Snippets/README.md:1]:
```suggestion
# .NET Cloud DLP API Samples
```

File: dlp/api/Snippets.Tests/InspectStringMultipleRulesTests.cs [hunk 2]
```diff
@@ -26,10 +26,9 @@ public InspectStringMultipleRulesTests(DlpTestFixture fixture)
             Fixture = fixture;
         }
 
-        [Fact(Skip = "https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/1159")]
```
Comment (amanda-tarafa) [inline dlp/api/Snippets.Tests/InspectStringMultipleRulesTests.cs:29]: You need to leave in [Fact] so that it's ran as a test.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
A minor change, thanks for fixing so quickly!

**Comment by pheonixblade9** [comment]
Please go ahead and merge this if it looks good.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**